fansi/shared/src/main/scala/fansi/Fansi.scala

```diff
@@ -679,10 +695,13 @@ object Back extends Category(offset = 12, width = 9){
     for(x <- 0 to 256)
```

Comment:
Aha! Here is the bug that was causing the `True` colors to be off by one. This, and the corresponding code above, should be `0 until 256` rather than `0 to 256`, as the range should *exclude* 256 itself. There is, in fact, no such color as `\u001b[48;5;256m`, as you can see here (https://upload.wikimedia.org/wikipedia/en/1/15/Xterm_256color_chart.svg)

Please help fix this! Then your code below can use 273 as the `True` offset as expected